packages/ui/src/primitives/tailwind/Modal/index.tsx

```diff
@@ -41,6 +41,7 @@ export interface ModalProps {
   children?: ReactNode
   submitLoading?: boolean
   showCloseButton?: boolean
+  hideHeaderCloseButton?: boolean
```

Comment:
you will need to make a separate modal using the individual components (maybe some modification will be needed)

there is also a back button present on the left side of the modal header
![image](https://github.com/user-attachments/assets/7fa88723-f108-4349-884e-c6115008f143)

Reply:
Modal with custom header:
![image](https://github.com/user-attachments/assets/8c59c09d-37ab-48cf-9f21-c681ceb46efe)

Reply:
Done